lib/engine/game/g_18_co.rb

```diff
@@ -0,0 +1,130 @@
+# frozen_string_literal: true
+
+require_relative '../config/game/g_18_co'
+require_relative 'base'
+require_relative 'company_price_50_to_150_percent'
+require_relative 'revenue_4d'
+
+module Engine
+  module Game
+    class G18CO < Base
+      register_colors(green: '#237333',
+                      red: '#d81e3e',
+                      blue: '#0189d1',
+                      lightBlue: '#a2dced',
+                      yellow: '#FFF500',
+                      orange: '#f48221',
+                      brown: '#7b352a',
+                      black: '#000000',
+                      pink: '#FF0099',
+                      purple: '#9900FF',
+                      white: '#FFFFFF')
+      load_from_json(Config::Game::G18CO::JSON)
+      AXES = { x: :number, y: :letter }.freeze
+
+      # DEV_STAGE = :beta
+
+      GAME_LOCATION = 'Colorado, USA'
+      GAME_RULES_URL = 'https://drive.google.com/open?id=0B3lRHMrbLMG_eEp4elBZZ0toYnM'
+      GAME_DESIGNER = 'R. Ryan Driskel'
+      GAME_INFO_URL = 'https://boardgamegeek.com/thread/1526930/18co-rock-stock-playtesting-files-available'
```

Comment:
I think you can make the GAME_INFO_URL as the other games, that is to the wiki page
(18MEX: https://github.com/tobymao/18xx/wiki/18MEX)
and from the wiki page you can possibly have the playtesting link.

Reply:
I have done this.